src/workingSets.ts

```diff
@@ -338,6 +339,13 @@ export class WorkingSetsProvider
     this.updateWorkspaceState()
   }
 
+  moveFile(workingSetItem: WorkingSetItem, direction: MoveDirection) {
+    if (!workingSetItem) return;
```

Comment:
Could you add curly braces around the `return` to keep `eslint` happy 😄? 
```
    if (!workingSetItem) { return; }
```

Reply:
Done